problems/arraysIntro.js

```diff
@@ -64,29 +89,45 @@ function arrayToString() {}
  * @param {Array} array2
  * @returns {string|Array}
  */
-function largerArray() {}
+function largerArray(array1, array2) {
+  if (array1.length > array2.length) {
+    return array1
+  }
+  else if (array2.length > array1.length) {
+    return array2
+  }
+  else if (array1.length === array2.length) {
+    return "They are the same size"
+  }
+}
 
 /**
  * Takes in an array and returns the middle element.
  * If the array is even in length return the middle two elements in their own array.
  * @param {Array} array
  * @returns {*|Array}
  */
-function middleElement() {}
+function middleElement(array) {
```

Comment:
Missing function. Try thinking about how you can look for the middle using the length property.